DOC-2098 - Karpenter Support Limitations (#7823)
spectrocloud/librarium: docs/docs-content/clusters/public-cloud/aws/architecture.md

![Karpenter node in Palette](/public-cloud_aws_architecture_managed-by-karpenter.webp)

#### Known Limitations

> - The following instance types are not supported by Palette and any non-supported instance types launched by Karpenter

Reviewer: ```suggestion
- Palette does not support the following instance types, and any non-supported instance types launched by Karpenter
```

  will not be displayed in the node pool view. Additionally, they will not be included in

>   [kch](../../../introduction/resource-usage-estimation.md) calculations.

Reviewer: ```suggestion
  [kCh](../../../introduction/resource-usage-estimation.md) calculations.
```
The linked page uses this format

>   - `t1`,`m1`,`c1`,`cc2`,`m2`,`cr1`,`cg1`,`i2`,`hs1`,`m3`,`c3`,`r3`

Reviewer: ```suggestion
  - `t1`, `m1`, `c1`, `cc2`, `m2`, `cr1`, `cg1`, `i2`, `hs1`, `m3`, `c3`, `r3`
```
Also, did you decide to present them in one line on purpose?

> - Karpenter-specific data is not shown for node pools managed by Karpenter, for example,

Author: I've managed to rewrite more actively.

> - Karpenter-specific data is not shown for node pools managed by Karpenter, for example,
>   [NodeClaims](https://karpenter.sh/preview/concepts/nodeclaims/) or
>   [Metrics](https://karpenter.sh/preview/reference/metrics/).

Author: ```suggestion
- Palette does not display Karpenter-specific data for node pools managed by Karpenter, such as [NodeClaims](https://karpenter.sh/preview/concepts/nodeclaims/) or [Metrics](https://karpenter.sh/preview/reference/metrics/).
```
